ui-modules/blueprint-composer/app/components/spec-editor/spec-editor.directive.js

```diff
@@ -254,6 +254,9 @@ export function specEditorDirective($rootScope, $templateCache, $injector, $sani
         }
         scope.nonempty = (o) => o && Object.keys(o).length;
         scope.defined = specEditor.defined = (o) => (typeof o !== 'undefined');
+        specEditor.isInstance = (x, type) => (typeof x === type);
+        specEditor.json = x => JSON.stringify(x, null, 2);
```

Comment:
No need for this, there is a pipe called `json` that you can use like this `{{ data | json }}`

Reply:
nice, i'll revert this